beacon-chain/blockchain/process_block.go

```diff
@@ -308,6 +309,26 @@ func (s *Service) onBlockBatch(ctx context.Context, blks []consensusblocks.ROBlo
 	return s.saveHeadNoDB(ctx, lastB, lastBR, preState, !isValidPayload)
 }
 
+func (s *Service) areSidecarsAvailable(ctx context.Context, avs das.AvailabilityStore, roBlock consensusblocks.ROBlock) error {
+	blockVersion := roBlock.Version()
+	block := roBlock.Block()
+	slot := block.Slot()
+
+	if version.Deneb <= blockVersion && blockVersion < version.Fulu {
```

Comment:
blockVersion < version.Deneb || blockVersion >= version.Fulu

another opportunity to use this

Reply:
Fixed in 41cf0ec59e458a503a6adc09856b0359b09b7321.